ts/data.ts

```diff
@@ -1,8 +1,29 @@
 /* exported data */
 
-let data = {
-  view: 'entry-form',
+interface JournalEntry {
+  entryId: number;
+  title: string;
+  photoUrl: string;
+  notes: string;
+}
+
+interface DataModel {
+  entries: JournalEntry[];
+  nextEntryId: number;
+}
+
+const data: DataModel = {
   entries: [],
-  editing: null,
   nextEntryId: 1,
 };
```

Comment:
We gave you a data object and you should have kept it. This is the data object you should have and you should update your data object and DataModel interface
```
let data = {
  view: 'entry-form',
  entries: [],
  editing: null,
  nextEntryId: 1,
};
```